src/core_plugins/console/public/.eslintrc

```diff
@@ -0,0 +1,36 @@
+---
+
+root: true
+extends: '../../../../.eslintrc'
+
+rules:
+  block-scoped-var: off
+  camelcase: off
+  curly: off
+  dot-location: off
+  dot-notation: off
+  eqeqeq: off
+  guard-for-in: off
+  indent: off
+  max-len: off
+  new-cap: off
+  no-caller: off
+  no-empty: off
+  no-extend-native: off
+  no-loop-func: off
+  no-multi-str: off
+  no-nested-ternary: off
+  no-proto: off
+  no-sequences: off
+  no-undef: off
+  no-use-before-define: off
+  one-var: off
+  quotes: off
+  space-before-blocks: off
+  space-in-parens: off
+  space-infix-ops: off
+  semi: off
+  strict: off
+  wrap-iife: off
+  no-var: off
+  prefer-const: off
```

Comment:
Just like the console-root eslintrc that is deleted in this PR, we can delete the specific lint configs once we've updated the code in those directories (public and api_server).